src/backend/distributed/shared_library_init.c

```diff
@@ -83,7 +83,9 @@ _PG_init(void)
 	if (!process_shared_preload_libraries_in_progress)
 	{
 		ereport(ERROR, (errmsg("Citus can only be loaded via shared_preload_libraries"),
-						errhint("Add citus to shared_preload_libraries.")));
+						errhint("Add citus to shared_preload_libraries in postgresql.conf"
+								"in master and worker nodes. Note that citus should be at"
```

Comment:
maybe something like
Add citus to shared_preload_libraries configuration variable in postgresql.conf in master and workers. Make sure citus is the first library in the list.